src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,19 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+
+    default double foodCosts(){
+        return 1000;
```

Comment:
not a good idea to return hardcoded values in default methods in an interface. Try to come up with another method